CommandLine/emake/EnigmaCallbacks.cpp

```diff
@@ -0,0 +1,92 @@
+#include "EnigmaCallbacks.hpp"
+
+#include <iostream>
+#include <pthread.h>
+#include <unistd.h>
+
+bool CallBack::_isOutputting = false;
+std::ifstream CallBack::_outFile;
+
+void* CallBack::OutputThread(void*)
+{
+    if (_outFile)
```

Comment:
Only the compiler stuff should be camel case, snake case is for GML functions only. I think that's pretty obvious.

You should generally stick to two spaces for indent and only worry about changing files that are a super messy mixture of tabs and spaces. If some file is all tabs, just leave it be.